### PR TITLE
Take Mx7 docs down

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/en/docs/appstore/use-content/modules/forgot-password.md [hunk 1]
```diff
@@ -33,7 +31,7 @@ If you already use the Forgot Password module in your Mendix 8 app, you can find
 
 The Forgot Password module has the following dependencies:
 
-* [Email Connector](/appstore/modules/email-connector/) – Versions of the Forgot Password module 3.x.x (for Mendix 7), below 4.1.0 (for Mendix 8), and 5.1.0 (for Mendix 9 and above) have a dependency on the deprecated [Email Module with Templates](https://marketplace.mendix.com/link/component/259/) module. If you are using Mendix 8 and above, Mendix recommends upgrading to the latest version using the instructions in the [Migrate from Email Module with Templates to Email Connector](#migrate-email) section below.
+* [Email Connector](/appstore/modules/email-connector/) – Versions of the Forgot Password module above 3.x.x, but below 4.1.0 (for Mendix 8), and 5.1.0 (for Mendix 9 and above) have a dependency on the deprecated [Email Module with Templates](https://marketplace.mendix.com/link/component/259/) module. If you are using Mendix 8 and above, Mendix recommends upgrading to the latest version using the instructions in the [Migrate from Email Module with Templates to Email Connector](#migrate-email) section below.
```
Comment (MariaShaposhnikova) [inline content/en/docs/appstore/use-content/modules/forgot-password.md:34]: @quinntracy , are you the owner of this doc? Could you please review my changes?

File: content/en/docs/appstore/use-content/modules/saml.md [hunk 2]
```diff
@@ -237,9 +237,6 @@ The following settings control user provisioning:
 
 * **Uses InCommon Federation Standard** – IdPs that use the InCommon standard often do not specify the assertion attributes. When following the InCommon standard, a fixed set of assertion attributes will be available to choose from later.
 * **Identifying Assertion** (aka "Principal Key") – Specifies which of the assertion attributes identifies the user name.
-
-    {{% alert color="info" %}}In Mendix 7, all user names passing through the SAML module are converted to lower-case before login and creation. We strongly advise you to convert all existing and new user names to lower-case as well.{{% /alert %}}
```
Comment (MariaShaposhnikova) [inline content/en/docs/appstore/use-content/modules/saml.md:241]: Is it fine to delete this?

File: content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ Follow these steps to export the Mendix components and store them on your local
 1. Start the Configuration Tool using the command `mxpc-cli registry-migration` to initiate registry migration mode.
 2. Select **Migration Type** to be *Export*.
 3. Select the **Main Components** and **Storage Provisioners** you want to export.
-4. Enter the version numbers of any Mendix Runtimes which you want to export. You can use wildcards and create a list of versions separated by commas. For example `8.12.*, 7.23.6.*` will export all patch versions of the Mendix 8.12 runtime and the published build of the Mendix 7.23.6 runtime.
+4. Enter the version numbers of any Mendix Runtimes which you want to export. You can use wildcards and create a list of versions separated by commas. For example `9.6.*, 10.12.3.*` will export all patch versions of the Mendix 9.6 runtime and the published build of the Mendix 10.12.3 runtime.
 
     {{< figure src="/attachments/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating/export.png" class="no-border" >}}
```
Comment (MariaShaposhnikova) [inline content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md]: @katarzyna-koltun-mx , is it a lot of work to change the image here?

File: content/en/docs/appstore/use-content/modules/forgot-password.md (hunk 1 above)
Comment (quinntracy) [reply content/en/docs/appstore/use-content/modules/forgot-password.md:34]: Looks good to me

File: content/en/docs/appstore/use-content/modules/saml.md (hunk 2 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/use-content/modules/saml.md:241]: Looks good to me!

File: content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md (hunk 3 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md]: I'll try to get a new one.

Comment (MariaShaposhnikova) [reply content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md]: As agreed, I will revert the change for now and will work on Mendix 7 instances in a separate PR, as this one gets too big.

**Pull request merged**